quick-start-aks.md

```diff
@@ -0,0 +1,198 @@
+
+Install `azure-cli` for your OS by following the official guide [Install az cli official](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli-macos). 
+
+## Login to Azure
+
+Run below command. Note that there are different way we can authenticate with Azure using Azure CLI refer [MS Docs Az login](https://learn.microsoft.com/en-us/cli/azure/authenticate-azure-cli)
+
+In this quick start guide we use the option `Sign in interactively` which would open a browser window to get the login credentials.
+
+``` bash
+az login
+```
+
+In case you have access to multiple Azure subscription set the account from CLI as below
+
+``` bash
+az account -s <Subscription name or Id>
+```
+## Create AKS Cluster for Dev/Test Purpose
+
+From the command line run below command to create a AKS cluster. Note that you need to create a new resource group to host the AKS resource.
+
+``` bash
+AKS_RESOURCE_GROUP="<Your Resource Group Name>"
+AKS_CLUSTER="initium-test-aks-cluster" # Set the name of the cluster as you require
+
+# Create Log Analytics Workspace
+AKS_MONITORING_LOG_ANALYTICS_WORKSPACE_ID=$(az monitor log-analytics workspace create \
+    --resource-group ${AKS_RESOURCE_GROUP} \
+    --workspace-name initium-test-aks-workspace \
+    --query id \
+    -o tsv)
+
+# Create AKS Cluster
+az aks create --resource-group ${AKS_RESOURCE_GROUP} \
+            --name ${AKS_CLUSTER} \
+            --enable-managed-identity \
+            --generate-ssh-keys \
+            --admin-username aksnodeadmin \
+            --node-count 1 \
+            --enable-cluster-autoscaler \
+            --min-count 1 \
+            --max-count 2 \
+            --network-plugin kubenet \
+            --node-vm-size Standard_DS3 \
+            --nodepool-labels nodepool-type=system nodepoolos=linux app=system-apps \
+            --nodepool-name systempool \
+            --nodepool-tags nodepool-type=system nodepoolos=linux app=system-apps \
+            --enable-addons monitoring \
+            --workspace-resource-id ${AKS_MONITORING_LOG_ANALYTICS_WORKSPACE_ID} \
+            --network-policy calico \
+            --vm-set-type VirtualMachineScaleSets \
+            --kubernetes-version 1.26.6
+
+# Get Kubernetes credentials
+az aks get-credentials --name ${AKS_CLUSTER}  --resource-group ${AKS_RESOURCE_GROUP} 
+
+```
+We recommend to use `Standard_DS3` as the VM Node size as the Initium workload needs more memory (at least 14 GiB)
+
+## Initium Platform Setup
+
+### Clone the platform repository
+
+```bash
+git clone https://github.com/nearform/initium-platform.git
+```
+
+### Install the required tooling
+
+```bash
+cd initium-platform
+make asdf_install
+```
+
+### Login to Azure CLI 
+
+Login using CLI as from above from the root of the `initium-platform` repo 
+
+>Note that at this point you would have a AKS Cluster ready to use.
+
+### Check Cluster access
+
+``` bash
+
+AKS_RESOURCE_GROUP="<Update your resource group>"
```

Comment:
Similar to the one above, let's move this to a separate code block so once I've set this I can copy and paste the rest without having to modify anything.

Reply:
Added